docs/src/architecture/mapper.md

```diff
@@ -1,9 +1,86 @@
-# The mapper
+# The tedge-mapper
 
-Or more appropriately the mappers,
-because there is a specific mapper for each cloud provider,
-a Cumulocity mapper, an Azure mapper.
-
-The purpose of these mappers is to translate
-messages written using the cloud-agnostic [Thin-Edge Json format](thin-edge-json.md),
+The tedge-mapper is a key concept to support multiple cloud providers.
+The purpose is to translate
+messages written using the cloud-agnostic [Thin Edge JSON format](thin-edge-json.md),
 into cloud-specific messages.
+
+tedge-mapper is composed of multiple cloud-specific mappers, such as Cumulocity mapper and Azure mapper.
+Each mapper is responsible for its dedicated cloud.
+> Note: The tedge-mapper contains the Cumulocity mapper only currently.
+
+A mapper subscribes the reserved MQTT topic `tedge/measurements` with the QoS level At Least Once.
+The messages that arrived in the mapper should be formed in the [Thin Edge JSON](thin-edge-json.md) format. 
+The mapper verifies whether the arrived messages are correctly formatted,
+in case the verification fails, the mapper publishes a corresponded error message
+on the topic `tedge/errors` with the QoS level At Least Once.
```

Comment:
```suggestion
on the topic `tedge/errors` with the QoS level 1.
```

Reply:
I would make explicit both the QoS level and the QoS semantics, as in "QoS level 1 (at least once)".

Reply:
Changed to QoS level 1 (at least once)